docs/extensions/FrameworkElementEx.md

```diff
@@ -0,0 +1,42 @@
+---
+title: FrameworkElement extentions
+author: ST-Apps
+ms.date: 01/05/2018
+description: FrameworkElementEx provides a simple way to bind to actual size for any FrameworkElement
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, FrameworkElement, extentions
+---
+
+# FrameworkElement extentions
+
+FrameworkElementEx provides a simple way to bind to actual size for any FrameworkElement
+
+## EnableActualSizeBinding
+
+The EnableActualSizeBinding property allows to enable/disable the binding for ActualHeight and ActualWidth
+
+## ActualHeight
+
+The ActualHeight property allows to bind to TargetObject's ActualHeight
+
+### Example
+
+```xaml
+    <Rectangle x:Name="TargetObject"
+               extensions:FrameworkElementEx.EnableActualSizeBinding="true"/>
+	...
+	<TextBlock Text="{Binding ElementName=TargetObject, Path=(extensions:FrameworkElementEx.ActualHeight)}" />
+```
+
+## ActualWidth
+
+The ActualWidth works exactly as ActualHeight.
```

Comment:
Just copy the same sentence for the Actual Height, it's clearer that way as someone may not read that part first.